cli/README.md

```diff
@@ -2,12 +2,23 @@
 
 ## Installation
 
-Build with `cargo build --bin oxide` and add the executable in `target/debug` to your `PATH`.
+You can find pre-built binaries for various platforms
+[here](https://github.com/oxidecomputer/oxide.rs/releases). Look for the most
+recent release whose version suffix matches the version of Oxide software
+currently deployed. For example, if you're running the `20240502.0` release,
+use `0.5.0+20240502.0`.
```

Comment:
There might not be anything to change here about this, but `20240502.0` is the API version but "v8" is the primary release version. Travis just asked me the other day whether there was a way to know in the console what release you're on. Not only is there no way to do that, but even if you did know you were on v8, there isn't a way to know that that corresponds to `20240502.0`, or vice versa. It's worth noting that there _is_ a note in the [v8 release notes](https://docs.oxide.computer/release-notes/system/8) about 0.5.0 being the relevant CLI version.

Like I said, probably a bigger problem we can't fix in the readme for the CLI, but this reminded me of the problem — something to mull over as we keep tweaking this stuff.

Reply:
My hope is that our docs would send you to the right CLI version and that the CLI itself will be able to point you to the right version based on a response from the server. We're going to have the generated client send the version as a header. The server response can include an indication if you're on a mismatched version (ie with work in dropshot / omicron)